docs/book/src/topics/managedcluster.md

```diff
@@ -361,6 +361,24 @@ spec:
   osDiskType: "Ephemeral"
 ```
 
+## AKS Node Pool KubeletDiskType configuration
+
+You can configure the `KubeletDiskType` value for each AKS node pool (`AzureManagedMachinePool`) that you define in your spec (see [here](https://learn.microsoft.com/en-us/rest/api/aks/managed-clusters/create-or-update?tabs=HTTP#kubeletdisktype) for the official AKS documentation). There are two options to choose from: `"OS"` or `"Temporary"`.
```

Comment:
I tried creating a cluster with a `spec.kubeletDiskType` `Temporary` and ran into the following error in the controller logs:
```
E1215 22:15:03.423111      71 controller.go:326]  "msg"="Reconciler error" "error"="error creating AzureManagedControlPlane default/aks-29206: failed to reconcile AzureManagedControlPlane service managedcluster: failed to create resource aks-29206/aks-29206 (service: managedcluster): Code=\"PreviewFeatureNotRegistered\" Message=\"Preview feature Microsoft.ContainerService/KubeletDisk not registered.\"" "AzureManagedControlPlane"={"name":"aks-29206","namespace":"default"} "controller"="azuremanagedcontrolplane" "controllerGroup"="infrastructure.cluster.x-k8s.io" "controllerKind"="AzureManagedControlPlane" "name"="aks-29206" "namespace"="default" "reconcileID"="aef62a22-1e54-4c34-b0de-ab7ff1a73761"
```

So it might be good to drop a hint to run this before this can be used:
```
az feature register --namespace Microsoft.ContainerService --name KubeletDisk
```

Reply:
Great catch! I actually ran into the exact same issue, so I think it'll be great to add to the docs.